osu.Game.Rulesets.Osu/OsuInputManager.cs

```diff
@@ -80,6 +80,12 @@ public enum OsuAction
         LeftButton,
 
         [Description("Right button")]
-        RightButton
+        RightButton,
+
+        [Description("Unscored Left button")]
+        UnscoredLeftButton,
+
+        [Description("Unscored Right button")]
+        UnscoredRightButton
```

Comment:
I was going to prevent these from being stored in replays (if they are right now? I actually haven't checked) but their definitions aren't available where I think they need to be. Future mods that use them may want their data stored, too, and currently due to checks of `DisplayScoredResult` -> returns everything works for this. Unless having these replays play in Stable is a goal I think leaving it with whatever it is doing now is fine.